hourly-tips/src/test/java/org/apache/flink/training/exercises/hourlytips/HourlyTipsTest.java

```diff
@@ -72,27 +84,39 @@ public void testMaxAcrossDrivers() throws Exception {
         TaxiFare tenFor1In2 = testFare(1, t(90), 10.0F);
```

Comment:
Can you add a couple of more test fares so that chances are higher that these are processed on different parallel instances (if the users' code supports parallel instances)?